lib/compat/wordpress-6.0/block-template-utils.php

```diff
@@ -15,8 +15,8 @@
  * @param string $path The path of the file in the theme.
  * @return Bool Whether this file is in an ignored directory.
  */
-function gutenberg_is_theme_directory_ignored( $path ) {
-	$directories_to_ignore = array( '.svn', '.git', '.hg', '.bzr', 'node_modules', 'vendor' );
+function gutenberg_is_theme_file_ignored( $path ) {
+	$directories_to_ignore = array( '.DS_Store', '.svn', '.git', '.hg', '.bzr', 'node_modules', 'vendor' );
```

Comment:
What do you think about using `$ignore_list`, a more general variable name here?

Reply:
Good idea, updated.